lib/inventory.ml

```diff
@@ -13,6 +13,7 @@
  *)
 (* Code to parse the XenSource inventory file *)
 
+open Stdext
```

Comment:
Generally I agree, but this only brings relatively unusually names modules into scope, no vars or types.